oap-server/server-library/library-client/src/main/java/org/apache/skywalking/oap/server/library/client/elasticsearch/ElasticSearchClient.java

```diff
@@ -286,6 +288,24 @@ public boolean isExistsIndex(String indexName) throws IOException {
         return client.indices().exists(request);
     }
 
+    public long getDocNumber(String indexName) throws IOException {
```

Comment:
You forget to remove this permnantly?

Reply:
please review the installer again. used in isExist method.

Reply:
I read, and don't know why. We are checking template and index, whether having doc matters?

Reply:
I think this is important for preserving historical versions of the data.